portfolio/src/main/webapp/script.js

```diff
@@ -12,31 +12,88 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+window.addEventListener('DOMContentLoaded', () => {
+  const navBar = document.getElementById('nav-bar');
+
+  Array.from(navBar.children).forEach(child => {
+    child.addEventListener('click', () => {
+      const targetId = child.getAttribute('data-target-id');
+      const targetElement = document.getElementById(targetId);
+      showContent(targetElement);
+    });
+  });
+});
+
+/** Displays the content title and text in the content container. */
+function showContent(element) {
+  const contentContainer = document.getElementById('content-container');
+
+  Array.from(contentContainer.children).forEach(child => {
+    child.classList.toggle('hidden', true);
+  });
+
+  element.classList.toggle('hidden', false);
+}
+
+showContent(document.getElementById("about-me"));
+
 /** Retrieves data from the /data page and displays it. */
 function getComments() {
-  fetch('/data').then(response => response.json()).then((json) => {
+  clearComments();
+  localStorage.setItem("commentAmount", document.getElementById("comment-amount").value);
+  commentAmount = localStorage.getItem("commentAmount");
+  fetch('/data?comment-amount=' + commentAmount).then(response => response.json())
```

Comment:
Nit: use a template string for readability

```javascript
`/data?comment-amount=${commentAmount}`
```

Reply:
Done.